### PR TITLE
add custom permission for dbcleanup plugin

### PR DESCRIPTION
**Pull request by pgvishnuram**
Remove unnecessary validation of default roles , add custom role for dbcleanup plugin and adds it to admin role

### CODE REVIEW

**Review by ephraimbuddy — APPROVED** [review]

File: dbcleanup_plugin.py [hunk 1]
```diff
@@ -323,24 +322,24 @@ def export_cleaned_records(
...
 class AstronomerDbcleanup(AppBuilderBaseView):
+    resource_name = "dbcleanup"
     default_view = "dbcleanup"
+    csrf_exempt = False
+    base_permissions = ["can_access_dbcleanup"]
+    # A bug in Flask-AppBuilder mandates us to have a value for method_permission_name
+    # before is_item_public filter can be used.
+    method_permission_name = {
+        "access_dbcleanup": "access_dbcleanup",
+    }
```
Comment (ephraimbuddy) [inline dbcleanup_plugin.py:333]: I don't think you need this though. It was needed in version check plugin because we used is_item_public here: https://github.com/astronomer/astronomer-airflow-version-check/blob/9c9754dfea36de88a9a4882aa016164cb98dfd16/astronomer/airflow/version_check/templates/update-available.html#L4

File: dbcleanup_plugin.py [hunk 2]
```diff
@@ -18,12 +18,11 @@
 from airflow.utils import db_cleanup, dates
 from airflow.utils.db_cleanup import config_dict
 from airflow.settings import conf
-from airflow.security import permissions
 from airflow.www import auth
 
 from .cloud_providers import ProviderFactory
 
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```
Comment (ephraimbuddy) [inline dbcleanup_plugin.py:25]: For me, it's better to make a separate PR for the release

**Review by rishkarajgi — APPROVED** [review]
